packages/asc-ui/src/components/Typography/TypographyStyle.ts

```diff
@@ -1,13 +1,18 @@
 import styled, { css, styledComponents, Theme } from '@datapunt/asc-core'
 import { em, margin } from 'polished'
-import { getTypographyFromTheme, color as getColor } from '../../utils'
+import { getTypographyFromTheme, color as getColor, color } from '../../utils'
```

Comment:
color and getColor are the same. Maybe rename the color util function to something like getColor to prevent the naming conflict with the color css prop?